site/ru/README.md

```diff
@@ -1,31 +1,38 @@
-# Не Официальный Перевод
+# Неофициальный перевод
 
-Вся информация в этом разделе переведена при помощи русскоговорящего Tensorflow
-сообщества на общественных началах. По скольку этот перевод не является
+Вся информация в этом разделе переведена с помощью русскоговорящего Tensorflow
+сообщества на общественных началах. Поскольку этот перевод не является
 официальным, мы не гарантируем что он на 100% аккуратен и соответствует
 [официальной документации на английском языке](https://www.tensorflow.org/?hl=en).
-Если у вас есть предложение как исправить этот перевод, мы будем очень рады
+
+1. Если у вас есть предложение как исправить этот перевод, мы будем очень рады
```

Comment:
We can't use lists in this section since it will throw off the formatting of the "note" callout box that is added. See the top of this page: https://www.tensorflow.org/tutorials/keras/basic_classification?hl=ru

Reply:
Sure, I saw the added notes :)

My intention is to eventually expand the readme so that it could get the contributors get up and running asap with a quick glance.

I’m fine with the steps being in a single note though, rolling those back too 👍

Reply:
Sounds good. Let's keep this disclaimer section short so I can copy-paste it into notebooks. if needed :)

I think expanding the readme as a Russian contributor guide sounds like a great idea.

Reply:
Yep, I will be giving it a glance once in a while since I kind of feeling like *owning* the translations in a product sense 😀